sdk/servicebus/Microsoft.Azure.ServiceBus/src/QueueClient.cs

```diff
@@ -445,6 +445,16 @@ public void RegisterMessageHandler(Func<Message, CancellationToken, Task> handle
             this.InnerReceiver.RegisterMessageHandler(handler, messageHandlerOptions);
         }
 
+        /// <summary>
+        /// Cancels the continuuous reception of messages without closing the underlying service bus connection and unregisters the message handler.
+        /// <remarks>Register a message handler first, using <see cref="RegisterMessageHandler(Func{Message, CancellationToken, Task}, Func{ExceptionReceivedEventArgs, Task})"/> 
```

Comment:
>remarks [](start = 13, length = 7)

Also add a remark mentioning the behavior when this is called multiple times

Reply:
The same in other clients

---
In reply to: [302802451](https://github.com/Azure/azure-sdk-for-net/pull/6360#discussion_r302802451) [](ancestors = 302802451)

Reply:
Also, looks like there is a difference in the xml doc between different files. Could you make sure all xmldocs of this method have the same comment

---
In reply to: [302802465](https://github.com/Azure/azure-sdk-for-net/pull/6360#discussion_r302802465) [](ancestors = 302802465,302802451)